src/styles/main.scss

```diff
@@ -1,3 +1,14 @@
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
 }
+
+.wrap {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
```

Comment:
do you really need this wrapper? I mean you can give the same styles to `calendar` block

Reply:
@Andriy-Shymkiv i can't move this styles to calendar block because of markup will be broken, i added margin: 0 auto but it isn't good idea, so which option i can use here?